src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,167 @@
+package wolox.training.models;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Book {
+
+    private String genre;
+
+    @NotNull
+    @Column(nullable = false)
+    private String author;
+
+    @NotNull
+    @Column(nullable = false)
+    private String image;
+
+    @NotNull
+    @Column(nullable = false)
+    private String title;
+
+    @NotNull
+    @Column(nullable = false)
+    private String subtitle;
+
+    @NotNull
+    @Column(nullable = false)
+    private String publisher;
+
+    @NotNull
+    @Column(nullable = false)
+    private String year;
+
+    @NotNull
+    @Column(nullable = false)
+    private String pages;
+
+    @NotNull
+    @Column(nullable = false, unique = true)
+    private String isbn;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```

Comment:
Great that you have chosen the sequential strategy. In Postgres, when GenerationType.AUTO is used, it generates an incremental ID shared between the schema tables. This happens only for this database engine. To generate independent IDs for each table, we use GenerationType.SEQUENCE.

But for this to work we must use it as follows:

```suggestion
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "BOOK_SEQ")            
    @SequenceGenerator(name = "BOOK_SEQ", sequenceName = "BOOK_SEQ")
```

Reply:
Oh I see, I didn't know that. Thanks